RELEASE

```diff
@@ -5,6 +5,75 @@
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
       /_/   /_/
 
+AppScale version 3.5.0, released March 2018
+Highlights of features/bugs in this release:
+
+ - Add a datastore viewer to the dashboard
+ - Improve lb responsiveness
+ - Transaction groomer
+ - Add GQL query to datastore viewer
+ - Service statistics
+ - Taskqueue service stats
```

Comment:
`- Service statistics` and `- Taskqueue service stats` are not visible yet, so I'd remove it.
Before next release we will probably merge https://github.com/AppScale/appscale/pull/2680 and then we will be able to say that ` - Add taskqueue service statistics to Hermes`.